check/check.go

```diff
@@ -161,8 +162,11 @@ func (c *Check) Run() {
 		glog.V(2).Info(errmsgs)
 	}
 
-	res := c.Tests.Execute(out.String())
-	if res {
+	// Todo: check result for nil
```

Comment:
May as well do the todo about checking for nil, no? Though see next comment - if you don't pass back a pointer to a struct and split out the bool and string, you won't need to do that check - another reason why it would be more succinct that way.